### PR TITLE
eased the local setup via docker-compose at db directory and updated …

### LINKED ISSUE #507: eased the local setup of the project
configured docker-compose.yml which was already inside package>db , now folks can head over to this folder , hit docker-compose up to start db and redis and use the connection strings in the envs and  finally can start the app... no, need to get instances from other platform.

### PR DESCRIPTION
**Pull request by PrakharSinha-09**
…the readme accordingly.

### PR Fixes:
- 1 easy setup of project via docker-compose, for starting the postgres db instance
- 2 no, need to pick the instance from anywhere else, follow readme.md for installation (mentioned in Windows user section)

Resolves #507 

### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

File: packages/db/docker-compose.yml [hunk 1]
```diff
@@ -13,7 +13,12 @@ services:
     volumes:
       - 100xdevs:/data/postgres
     restart: unless-stopped
-
+  
+  redis:
```
Comment (devsargam) [inline packages/db/docker-compose.yml:17]: Can you remove redis portion

File: packages/db/.env.example [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 DATABASE_URL="postgresql://johndoe:randompassword@localhost:5432/mydb?schema=public"
 
-REDIS_URL=""
+REDIS_URL="redis://localhost:6379"
```
Comment (devsargam) [inline packages/db/.env.example:4]: same as above

File: README.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ cd daily-code
 yarn install
 ```
 
-> Setup DB 
+> Setup DB & Redis
```
Comment (devsargam) [inline README.md:18]:
```suggestion
> Setup DB 
```

**Review by devsargam — CHANGES_REQUESTED** [review]

File: packages/db/docker-compose.yml (hunk 1 above)
Comment (PrakharSinha-09) [reply packages/db/docker-compose.yml:17]: there was redis url written already written in env.example.. I thought redis is also being used.... No worries let me remove it!

**Comment by PrakharSinha-09** [comment]
hey sargam, I've made new commit fixing what you said!

**Pull request merged**